MountSionCBSSecondary/SchoolWebsite
# added wse report

# Description

WSE Report Added 

## What

WSE Report Added to `About` -->  `Inspection Reports`

## Why

Parents can access WSE Report on website

## Verification Steps

http-server, go to About --> Inspection Reports and ensure that WSE Report 2015 is there along with its contents.


# Checklist:
